src/config.rs

```diff
@@ -199,21 +199,32 @@ pub enum Config {
     /// The primary email address. Also see `SecondaryAddrs`.
     ConfiguredAddr,
 
+    /// List of configured IMAP servers as a JSON array.
+    ConfiguredImapServers,
+
     /// Configured IMAP server hostname.
+    ///
+    /// This is replaced by `configured_mail_servers` for new configurations.
```

Comment:
`configured_imap_servers` or even `ConfiguredImapServers`. The same for the doc comments below